yang/ietf-schedule.yang

```diff
@@ -540,4 +555,71 @@ module ietf-schedule {
         "Defines a list of exceptions for recurrence.";
     }
   }
+  
+  grouping schedule-status {
+    description
+      "This grouping is defined to identify common properties of 
+       scheduling status.";
+    leaf schedule-id {
+      type string;
+      description
+        "The schedule identifier that globally identifies a schedule.";
+    }
+    leaf schedule-type {
+      type identityref {
+        base schedule-type;
+      }
+      description
+        "The schedule type";
+    }    
+    leaf last-update {
+      type yang:date-and-time;
+      description
+        "The timestamp that the schedule is last updated.";
+    }
+    leaf time-zone-identifier {
+      type sys:timezone-name;
+      description
+        "Indicates the referenced time zone identifier of the schedule.";
+    }
+    leaf version {
+      type uint16;
+      description
+        "The version number of the schedule.";
+    }
+    leaf state {
```

Comment:
Move this one to be defined as identity as the status may have other values such as, enabled but not in use (because of a conflict), etc.

Reply:
Can we exhaust all states here?